books/dummy/bake

```diff
@@ -7,13 +7,20 @@ gemfile do
   gem 'slop', '4.8.2'
 end
 
+recipe = Kitchen::BookRecipe.new(book_short_name: :dummy) do |doc|
+  include Kitchen::Directions
+
+  book = doc.book
+  book.search('div').first.replace_children(with: 'Hello, world!')
```

Comment:
CE may be using `dummy` and expecting no output.  I'd check with them since I think they made this file and see if they are ok with a different behavior.

Reply:
checked, sounds like it's okay!